src/Razor/src/Microsoft.CodeAnalysis.Razor/BindTagHelperDescriptorProvider.cs

```diff
@@ -315,6 +336,24 @@ private List<TagHelperDescriptor> CreateElementBindTagHelpers(List<ElementBindDa
                     // WTE has a bug 15.7p1 where a Tag Helper without a display-name that looks like
                     // a C# property will crash trying to create the toolips.
                     a.SetPropertyName(name);
+
+                    a.BindAttributeParameter(parameter =>
+                    {
+                        parameter.Name = "format";
+                        parameter.TypeName = typeof(string).FullName;
+                        parameter.Documentation = ComponentResources.BindTagHelper_Element_Format_Documentation;
+
+                        parameter.SetPropertyName(formatName);
+                    });
+
+                    a.BindAttributeParameter(parameter =>
+                    {
+                        parameter.Name = "event";
+                        parameter.TypeName = typeof(string).FullName;
+                        parameter.Documentation = "Element Event documentation";
```

Comment:
Inconsistent.  Don't really care but makes me question if there's a ComponentResources.BindTagHelper_Element_Event_Documentation

Reply:
No I was planning to add that. Missed it